roles/docker_image_build/tasks/docker_build.yml

```diff
@@ -18,8 +18,10 @@
     push: no
     build:
       path: "{{ docker_build_directory.path }}"
-      # explicitly fetch updates to the FROM image, so things like "latest" continue to be up to date
-      pull: yes
+      # enable skipping of pulling new FROM images
+      # https://www.docker.com/increase-rate-limits
+      # left to use the default docker_image default, which is *currently* no, but will change in community.general 2.0
+      pull: "{{ docker_build_pull | default(omit) }}"
```

Comment:
so a question... do we want to default this variable to `yes` so that we introduce this change passively?

Reply:
My comment was actually wrong, it currently defaults to `yes`.  I've updated the PR to fix that.

As for defaulting to a specific value, I did consider that, but chose to let the ansible module use its own default, unless the role _should_ have an opinion regarding the default behavior.  I'm not sure this role should have such an opinion, but I'm listening if you disagree.

Reply:
No disagreement.... I was just concerned with a default yes becoming a default no without a major upgrade. But if the underlying module is default yes, and a major version upgrade makes that change, that's totally cool.